be/src/vec/olap/vertical_merge_iterator.h

```diff
@@ -190,14 +190,14 @@ class VerticalMergeIteratorContext {
     size_t _ori_return_cols = 0;
 
     // segment order, used to compare key
-    uint32_t _order = -1;
+    uint32_t _order = 0;
 
-    uint32_t _seq_col_idx = -1;
```

Comment:
`_seq_col_idx` change to int32_t is better? align with the type in schema

Reply:
fixed